src/components/CreateMessage.tsx

```diff
@@ -0,0 +1,82 @@
+import { Button, Form, Input, InputNumber, Typography } from "antd";
+import { useState } from "react";
+import { addMessage } from "../services/chain/apis/extrinsic";
+import { SchemaProps } from "../services/types";
+import { staticSchema } from "./RegisterSchema";
+
+const {Text} = Typography;
+
+const CreateMessage = (props: SchemaProps): JSX.Element => {
```

Comment:
I suggest always having distinct props per component.
```suggestion
const CreateMessage = (props: { schema: SchemaDetail }): JSX.Element => {
```

Or

```suggestion
interface CreateMessageProps {
  schema: SchemaDetails;
}
const CreateMessage = (props: CreateMessageProps): JSX.Element => {
```